File: java/app/src/main/java/tw/com/ecpay/paymentgatewaykit/example/main/model/GatewaySDKModel.java
package tw.com.ecpay.paymentgatewaykit.example.main.model;

import androidx.databinding.ObservableField;

public class GatewaySDKModel {

    public final ObservableField<String> description = new ObservableField<>();

    public final ObservableField<String> sdkVersion = new ObservableField<>();

    public final ObservableField<String> token = new ObservableField<>();

    public final ObservableField<Integer> languageSelectedPosition = new ObservableField<>();

    public final ObservableField<Boolean> useResultPageSwitch = new ObservableField<>();

    public final ObservableField<String> xmlMerchantID = new ObservableField<>();

    public final ObservableField<String> titleBarBackgroundColor = new ObservableField<>();
}

File: kotlin/app/src/main/kotlin/tw/com/ecpay/paymentgatewaykit/example/main/model/GatewaySDKModel.kt
package tw.com.ecpay.paymentgatewaykit.example.main.model

import androidx.databinding.ObservableField

class GatewaySDKModel {

    val description = ObservableField<String>()

    val sdkVersion = ObservableField<String>()

    val token = ObservableField<String>()

    val languageSelectedPosition = ObservableField<Int>()

    val useResultPageSwitch = ObservableField<Boolean>()

    val xmlMerchantID = ObservableField<String>()

    val titleBarBackgroundColor = ObservableField<String>()
}